File: Javascript/8-Kyu/What's-the-real-floor?.js
// What's the real floor?
function getRealFloor(n) {

    if ( n <= 0 ) {
        return n;
    } else if ( 13 < n ) {
        return n - 2;
    } else {
        return n - 1;
    }

};

// Tests
console.log(getRealFloor(1));  // --> 0
console.log(getRealFloor(5));  // --> 4
console.log(getRealFloor(15));  // --> 13

File: Python/8-Kyu/What's-the-real-floor?.py
# What's the real floor?
def get_real_floor(n):
    
    return n if n <= 0 else n - 1 if 0 < n < 13 else n - 2

# Tests
print(get_real_floor(1))  # --> 0
print(get_real_floor(5))  # --> 4
print(get_real_floor(15))  # --> 13
